tests/tools/test_sa.py

```diff
@@ -33,12 +31,28 @@ def test_glm(self):
         assert -0.06523274132877163 == pytest.approx(age)
 
     def test_kmf(self):
-        import ehrapy as ep
-
         adata = ep.dt.mimic_2(encoded=False)
         adata[:, ["censor_flg"]].X = np.where(adata[:, ["censor_flg"]].X == 0, 1, 0)
         kmf = ep.tl.kmf(adata[:, ["mort_day_censored"]].X, adata[:, ["censor_flg"]].X)
 
         assert isinstance(kmf, lifelines.fitters.kaplan_meier_fitter.KaplanMeierFitter)
         assert len(kmf.durations) == 1776
         assert sum(kmf.event_observed) == 497
+
+    def test_anova_glm(self):
+        adata = ep.dt.mimic_2(encoded=False)
+        formula = "day_28_flg ~ age"
+        var_names = ["day_28_flg", "age"]
+        family = "Binomial"
+        age_glm = ep.tl.glm(adata, var_names, formula, family, missing="drop", ascontinus=["age"])
+        age_glm_result = age_glm.fit()
+        formula = "day_28_flg ~ age + service_unit"
+        var_names = ["day_28_flg", "age", "service_unit"]
+        ageunit_glm = ep.tl.glm(adata, var_names, formula, family="Binomial", missing="drop", ascontinus=["age"])
+        ageunit_glm_result = ageunit_glm.fit()
+        dataframe = ep.tl.anova_glm(
+            age_glm_result, ageunit_glm_result, "day_28_flg ~ age", "day_28_flg ~ age + service_unit"
+        )
+
+        assert len(dataframe) == 2
```

Comment:
Can we also test the actual value ranges aka the results? With some small deviations being allowed.